packages/workflow-core/src/lib/workflow-runner.ts

```
@@ -772,7 +772,7 @@ export class WorkflowRunner {
 
   private async __invokeApiPlugin(apiPlugin: HttpPlugin, additionalContext?: AnyRecord) {
     // @ts-expect-error - multiple types of plugins return different responses
-    const { callbackAction, responseBody, error } = await apiPlugin.invoke?.(
+    const { callbackAction, responseBody, requestPayload, error } = await apiPlugin.invoke?.(
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)